tutor/src/components/annotations/summary-page.jsx

```diff
@@ -87,8 +88,8 @@ class FilterWidget extends React.Component {
             checked={this.props.selectedChapters.length === this.parsedChapters.length}
           />
           {
-            this.parsedChapters.map((ch) => (
-              <CheckBoxEntry key={ch}
+            this.parsedChapters.map((ch, i) => (
+              <CheckBoxEntry key={i}
```

Comment:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
yeah, I know but I was getting react warnings about duplicate keys if there were multiple annotations per chapter :( In this case I think this is the best we can do.